<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modificar Receta</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>351</width>
      <height>139</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="idDisplayLabel">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="treatmentLabel">
       <property name="text">
        <string>Tratamiento:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="treatmentInput"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="startDateLabel">
       <property name="text">
        <string>Fecha de Inicio:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="startDateInput"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="idDisplay">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ID de la Receta</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="volverButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guardarButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar Cambios</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>351</width>
      <height>105</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="finalizedLabel">
       <property name="text">
        <string>Finalizada:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="finalizedInput"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="pacientLabel">
       <property name="text">
        <string>Paciente:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="pacientComboBox"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>